tests/controllers/login.test.js

```diff
@@ -78,13 +88,14 @@ describe('POST /signin', () => {
     it('redirects to "/login" path', () => {
       oauthApi.authenticate.rejects({ response: { status: 503 } })
 
-      return request(app)
+      return request
```

Comment:
took me a while, but just doing `request(app)` doesn't keep state between redirects so we lose our flash messages